docs/en/v0.4/user-guide/operations/monitor-&-alert.md

```diff
@@ -1 +1,122 @@
-# Monitor & alert
+# Monitor GreptimeDB
+
+To collect GreptimeDB metrics using Prometheus
+
+## Promethues Configuration
+
+Write a Prometheus configuration file and save it as prometheus.yml:
+```
+global:
+  scrape_interval: 15s 
+
+scrape_configs:
+  - job_name: 'greptimedb'
+    static_configs:
+      - targets: ['localhost:4000']
+```
+
+## Start GreptimeDB and Prometheus
+### Binary
+
+Install GreptimeDB according to the [documentation](https://docs.greptime.com/getting-started/try-out-greptimedb#binary)
+
+Visit [the official documentation for Prometheus](https://prometheus.io/download/) to download the binary. Afterward, execute the following command:
+
+```
+./prometheus --config.file=prometheus.yml
+```
+
+Access Prometheus by entering `localhost:9090` in your web browser.
+
+
+### Docker
+
+Install GreptimeDB according to the [documentation](https://docs.greptime.com/getting-started/try-out-greptimedb#docker)
+
+Run Prometheus:
+```
+docker run \
+  -p 9090:9090 \
+  -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
+  prom/prometheus
+```
+
+### Kubernetes
+
+Install [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+```
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus prometheus-community/kube-prometheus-stack
+```
+
+Use [gtctl](https://docs.greptime.com/user-guide/operations/gtctl) to install GreptimeDB cluster:
+```
+gtctl cluster create mycluster -n default \
+  --set cluster.prometheusMonitor.enabled=true \
+  --set cluster.prometheusMonitor.path="/metrics" \
+  --set cluster.prometheusMonitor.port="http" \
+  --set cluster.prometheusMonitor.interval="30s" \
+  --set cluster.prometheusMonitor.honorLabels=true \
+  --set cluster.prometheusMonitor.labelsSelector.release="prometheus"
+```
+
+
+## Metrics Detail
+
```

Comment:
Add a description that this part of information is subject to change. To find update-to-date information, please check the output of `curl http://<host>:<port>/metrics`

It will be difficult to keep it update to date.